src/packagedcode/dockerfileLABEL.py

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# ScanCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/scancode-toolkit for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+
+
+import io
+from pathlib import Path
+from dockerfile_parse import DockerfileParser  
+from packagedcode import models
+from packagedcode import utils
+
+
+class DockerfileHandler(models.DatafileHandler):
+    datasource_id = 'dockerfile'
+    default_package_type = 'docker-image'
+    path_patterns = ('Dockerfile', 'containerfile', '*.dockerfile')  
+    description = 'Dockerfile (OCI) metadata handler'
+    documentation_url = 'https://docs.docker.com/engine/reference/builder/'

```

Comment:
Shouldn't this be https://github.com/opencontainers/image-spec/blob/main/annotations.md ? Again this is not really about the usual dockerfile contents but only the OCI labels